src/mito2/src/engine.rs

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Mito region engine.
+
+#[cfg(test)]
+mod tests;
```

Comment:
Why put it in another directory and file? Looks like they are not very long

Reply:
It will become very long after porting all tests from our storage crate. (create/open/write/flush/compaction.......)